jbpm-persistence-jpa/src/main/java/org/jbpm/persistence/processinstance/JPAProcessInstanceWorkItemManager.java

```diff
@@ -0,0 +1,17 @@
+package org.jbpm.persistence.processinstance;
+
+import org.drools.core.common.InternalKnowledgeRuntime;
+import org.drools.persistence.jpa.processinstance.JPAWorkItemManager;
+
+/**
+ * This class is a jBPM-specific work item manager that should be used whenever
+ * 1. jBPM is in the classpath
+ * 2. and persistence is *not* being used.
```

Comment:
Hi, Marco,

I am guessing that since this is *JPA*ProcessInstanceWorkItemManager, it should be: 2. and persistence is being used.

Marian